### PR TITLE
Update .env.sample

### PR DESCRIPTION
**Pull request by shogochiai**

### CODE REVIEW

**Comment by shogochiai** [comment]
close #279

File: .env.sample [hunk 1]
```diff
@@ -2,6 +2,7 @@ ROOTCHAIN_ENDPOINT=http://127.0.0.1:8545
 ROOTCHAIN_ADDRESS=0xb9A219631Aed55eBC3D998f17C3840B7eC39C0cc
 OWNERSHIP_PREDICATE=0x9FBDa871d559710256a2502A2517b794B482Db40
 OPERATOR_PRIVATE_KEY=0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3
+OWNERSHIP_PREDICATE=0x9FBDa871d559710256a2502A2517b794B482Db40
```
Comment (syuhei176) [inline .env.sample]: duplicated in line 3 and 5

**Comment by shogochiai** [comment]
@syuhei176 I put `.env.sample` under `packages/operator` though, if `packages/operator/.env` is being autogenerated by lerna or something, please feed me back the fact.

**Comment by syuhei176** [comment]
> @syuhei176 I put `.env.sample` under `packages/operator` though, if `packages/operator/.env` is being autogenerated by lerna or something, please feed me back the fact.

no problem now.

**Pull request merged**